pravega/flink-connectors
# Issue 525 introduce dependency sorting in the checkstyle configurations in r0.10 flink1.11

Signed-off-by: zhongle.wang <zhongle_wang@dell.com>

**Change log description**

Noticed that some imports are not sorted in the code base, this may be introduced from constant PRs, we need to fix this in all the 0.10 branches and add checks to avoid further similar issues.

The desired ordering of the `import`.
* third party packages
* blank line
* javax.*
* java.*
* blank line
* static import

**Purpose of the change**

Fixes #525 

**What the code does**

* Change the order of `import`.
* fix the nonexisting RuntimeContextInitializationContextAdapters

**How to verify it**

* all files have the right `import` ordering.
* `./gradlew checkstyleMain checkstyleTest` should pass.
* `./gradlew clean build` should pass.
